.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -0,0 +1,166 @@
+      - ${{ if eq(parameters.datapath, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Datapath
+            name: datapath
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '(.*).Networking.Granular|(.*)kubernetes.api'
+            ginkgoSkip: 'SCTP|hostNetwork'
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 10
+      - ${{ if eq(parameters.netpol, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Network Policy
+            name: netpol
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '(.*).Netpol'
+            ginkgoSkip: 'SCTP'
+            os: ${{ parameters.os }}
+            processes: 1
+            attempts: 0
+      - ${{ if eq(parameters.dns, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: DNS
+            name: dns
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-network\].DNS.should'
+            ginkgoSkip: 'resolv'
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if eq(parameters.container, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Container Conformance
+            name: container
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-node\].Pods'
+            ginkgoSkip: 'Slow'
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 2
+      - ${{ if eq(parameters.discovery, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Resource Discovery
+            name: discovery
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-api-machinery\].Discovery'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if eq(parameters.liveness, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Liveness Probes
+            name: liveness
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-node\].Probing'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 10
+      - ${{ if eq(parameters.portforward, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Kubectl Portforward
+            name: portforward
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-cli\].Kubectl.Port'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
```

Comment:
didnt we discuss on removing some of these as its not required?

Reply:
The tests being run are controlled by what parameters are passed into it. 
```
          datapath: true
          dns: true
          portforward: true
          loadBalancer: true
          hybridWin: true
```
As I documented what these tests are doing, even the ones that we are not using, I saw no need to delete them.

Reply:
can we remove if we are not using it and add it back if required?